src/v0/destinations/marketo_static_list/transform.js

```diff
@@ -74,14 +74,14 @@ const processEvent = (input) => {
     (Array.isArray(toAdd) && toAdd.length > 0) ||
     (Array.isArray(toRemove) && toRemove.length > 0)
```

Comment:
Do this opposite 

`(!Array.isArray(toAdd) || !toAdd.length > 0) &&
    (!Array.isArray(toRemove) || !toRemove.length > 0)`
    Throw Error ..

Fail first